src/components/Sidebar.js

```diff
@@ -178,9 +178,12 @@ const Sidebar = ({ isOpen, toggle }) => {
 				<Trans
 					i18nKey="sidebar.poweredBy"
 					components={{
-						// eslint-disable-next-line jsx-a11y/anchor-has-content
 						docLinkWalletGithub: <a
-							href="https://github.com/wwWallet" rel="noreferrer" target='blank_' className="underline"
+							href="https://github.com/wwWallet"
+							rel="noreferrer"
+							target='blank_'
+							className="underline"
+							aria-label={t('sidebar.poweredbyAriaLabel')}
```

Comment:
I think `sidebar.poweredbyAriaLabel` is undefined in the translation file?

Reply:
Actually, I mistakenly added `sidebar.poweredbyAriaLabel` in my previous commit (2b6398200301e3245e62a67182176d183acaf1e2).

Reply:
Ah, I see! :smile: